File: 백준/Bronze/4641. Doubles/Doubles.py
while True:
    cnt = 0
    ls = list(map(int, input().split()))

    if ls[0] == -1:
        break
    for n in ls:
        for m in ls:
            if n == m:
                continue
            if n * 2 == m:
                cnt += 1
                break
    print(cnt)

File: 백준/Bronze/4641. Doubles/Doubles.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? 'dev/stdin' : 'run/input.txt';
const input = fs.readFileSync(filePath).toString().trim().split('\n');

for (ls of input) {
  if (ls === '-1') break;
  console.log(
    ls
      .split(' ')
      .map(Number)
      .filter((v, _, arr) => v !== 0 && arr.includes(v * 2)).length,
  );
}
